05_syntax_lab/01.py

```diff
@@ -1,5 +1,7 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+############### Part 1
+print('Please enter 10 numbers in one raw, separated by a space')
+numbers = input()
+print('These are the numbers:')
+print(numbers)
+print('The biggest value is:')
+print(max(numbers))
```

Comment:
OK, I understand from @ynonp it is ok for you to use Python 3. I'll check your solutions according.

In that case ```input``` is ok, though I think for the sake of this exercises it would be better to expect each number on a separate line.